spec/controllers/collection_items_controller_spec.rb

```diff
@@ -29,4 +103,26 @@
       end
     end
   end
+
+  describe "#destroy" do
+    before(:each) do
+      @collection = FactoryGirl.create(:collection)
+      @approvedwork = FactoryGirl.create(:work)
+      @approvedwork.add_to_collection(@collection) && @approvedwork.save
+    end
+
+    context "destroy" do
+      let(:owner) { @collection.owners.first.user }
+      render_views
+
+      it "removes things" do
+        @approvedworkitem = CollectionItem.find_by_item_id(@approvedwork.id)
+        fake_login_known_user(owner)
+        delete :destroy, id: @approvedworkitem.id
+        expect(response).to have_http_status(:redirect)
+        expect(flash[:notice]).to include "Item completely removed from collection"
```

Comment:
This is checking for a flash message and a redirect, so can we use one of our redirect expectations helpers here? #2733 adds some new ones if the existing ones don't work.